src/commands/cmd_bit.cc

```diff
@@ -121,7 +121,7 @@ class CommandBitCount : public Commander {
     if (args.size() == 5) {
       if (util::EqualICase(args[4], "BYTE")) {
```

Comment:
Can you explicitly set the value of the variable? This will make the code more readable.